api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -532,17 +543,8 @@ public void run() {
 		        String.valueOf(passwordField.getPassword()) });
 	}
 	
-	/**
-	 * Override this method to turn on/off the in-memory database. The default is to use the
-	 * in-memory database. When this method returns false, the database defined by the runtime
-	 * properties is used instead
-	 * 
-	 * @return true/false whether or not to use an in memory database
-	 */
-	public Boolean useInMemoryDatabase() {
-		return !"false".equals(System.getProperty("useInMemoryDatabase"));
-	}
```

Comment:
Why change position of this method?

Reply:
had changed it so that it can be overridden to turn off the in memory DB hence sitting on the extern al DB credentials in the runtime properties

Reply:
is it what could have caused the travis failure? coz i got a build success while compiling

Reply:
> had changed it so that it can be overridden to turn off the in memory DB hence sitting on the extern al DB credentials in the runtime properties

Can't the method be overridden from the original position. Asn what's the logic behind this!